module/x/gravity/keeper/ethereum-originated.go

```diff
@@ -0,0 +1,42 @@
+package keeper
```

Comment:
What's this all about?

Reply:
didn't want to make keeper.go any longer decied to try expanding it like we did for batch.go, not sure I like the pattern.